### DISCUSSION TITLE
Can't managed to install .deb of alpha release

### DISCUSSION THREAD

**Discussion by uaygunn** [gravitational/teleport#20752] [Q&A] [answered]
Hello guys and thank you all contributors for excellent work. I appriciate it.

For my question i wanted to try alpha release for non-ad desktop access but i could not make it work. I tried to install it from .deb package with "dpgk-i" it doesn't show any error but at the end says that command not found. I know it is a dumb question but how can i manage it to work ? Here is the dpkg results.

dpkg -i teleport-connect_12.0.0-alpha.1_amd64.deb
(Reading database ... 131362 files and directories currently installed.)
Preparing to unpack teleport-connect_12.0.0-alpha.1_amd64.deb ...
Unpacking teleport-connect (12.0.0-alpha.1) over (12.0.0-alpha.1) ...
teleport-connect: Upgrade detected, skipping symlink operations
Setting up teleport-connect (12.0.0-alpha.1) ...
/var/lib/dpkg/info/teleport-connect.postinst: line 13: update-desktop-database: command not found
Processing triggers for hicolor-icon-theme (0.17-2) ...

And it ends doesnt show anything afterwards.

**Comment by webvictim** [comment, marked as answer]
`teleport-connect` is a graphical client for connecting to Teleport clusters.

It's a different package to `teleport` itself which contains the binaries that are needed to run Teleport as a service.

Non-AD Windows desktop access is still being worked on and requires extra steps. I would wait for either the Teleport 12 release candidate or the actual release to happen so that the proper documentation and steps are available before testing.
